src/etc/rulesDTD.dtd

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this, where is it needed for? Tests? Then make it a test resource otherwise a normal resource?